gen/internal/tests/thrift/hyphenated_file.thrift

```diff
@@ -0,0 +1,5 @@
+include "./non_hyphenated.thrift"
```

Comment:
It took me a second to understand why this was called `hyphenated_file` without a hyphen.

We should add a comment.

```suggestion
// This file is named hyphenated_file to possibly conflict with the code
// generated from hyphenated-file.

include "./non_hyphenated.thrift"
```